app/views/homepage/_links_and_search.html.erb

```diff
@@ -34,7 +34,14 @@
       </div>
       
       <div class="govuk-grid-column-one-half homepage-search">
-        <form action="/search" method="get" role="search">
+        <form 
+          action="/search" 
+          method="get" 
+          role="search" 
+          data-module="ga4-form-tracker"
+          data-ga4-form-include-text
+          data-ga4-form='{"event_name": "search","type": "home page","url": "/search/all","section": "Search","action": "search"}'
```

Comment:
The `action` correlates with the button text on the form, which has an uppercase 'S' i.e. `Search`, which is how I implemented the [tracking on the header search box](https://github.com/alphagov/govuk_publishing_components/pull/3269/files#diff-156fd49dc93ceb491930d123a9d47c288efc722b989c5db4f424438e4968d10eR266), should probably be consistent.